src/components/Image/index.tsx

```diff
@@ -45,7 +47,18 @@ class Image extends BaseComponent<ImageProps> {
     this.throttledLazyLoadHandler = throttle(this.lazyLoadImage, 150);
   }
 
+  validateOpacity(opacity: string | number | undefined) {
```

Comment:
For this validation you could use the PropertyChecker located in `/src/components/utils/`.